transfuse-core/pom.xml

```diff
@@ -35,8 +35,8 @@
         <dependency>
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
-            <version>2.4</version>
-            <scope>test</scope>
```

Comment:
I'm hesitant to bring commons-io in as a dependency.  Looking at the source for `WriterOutputStream`, it seems pretty simple.  Why not just bring that one class in and possibly pair it down to the essential elements?

Reply:
you are right, I just put it into a new class.